projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -1456,3 +1460,13 @@ protected static function build_sprintf( $args ) {
 function publicize_calypso_url() {
 	return Redirect::get_url( 'calypso-marketing-connections', array( 'site' => ( new Status() )->get_site_suffix() ) );
 }
+
+/**
+ * Adds support for the post-thumbnails feature, regardless of underlying theme support.
+ *
+ * This ensures the featured image UI appears in the editor, allowing the user to
+ * explicitly set an image for their social media post.
+ */
+function add_theme_post_thumbnails_support() {
+	add_theme_support( 'post-thumbnails', array( 'post' ) );
```

Comment:
Instead of only adding it to the post editor, should we aim to add support to any post type that supports Publicize?

See this for prior art on this:
https://github.com/Automattic/jetpack/blob/60ff8e8f3400ab8e8bf7c68f1377e506d6f1cfd9/projects/plugins/jetpack/_inc/lib/core-api/wpcom-fields/post-fields-publicize-connections.php#L66-L67

Reply:
Thanks! This was my intention but I thought that only `'post'` types supported Publicize. `get_post_types_by_support()` is actually what I want 👍

Reply:
I've changed the hardcoded array to `get_post_types_by_support( 'publicize' )`, however I've also had to change the priority of the hook that adds `'publicize'` support to the `'post'` post type since they both happen during the `init` action.

https://github.com/Automattic/jetpack/blob/958e0b05e930e8df6e90c3f4ef61b9b6ba0836b9/projects/packages/publicize/src/class-publicize-base.php#L222